### ISSUE TITLE
GMFCS level III

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#7280]
**Preferred term label:**
GMFCS level III

**Synonyms**
The Gross Motor Function Classification System level III

**Definition (free text, please give PubMed ID)**
Children with cerebral palsy classified under GMFCS level III can walk using a hand-held mobility device in most indoor settings; they may climb stairs holding onto a railing with supervision or assistance; they use wheeled mobility when travelling long distances and may self-propel for shorter distances (PMID: 27188686).

**Parent term (use hpo.jax.org/app)**
Functional motor deficit HP:0004302

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by pnrobinson** [comment]
As mentioned, I think these terms are not good for HPO because they bundle a number of different items. For HPO, we want to capture the detailed data.
